### PR TITLE
update neuro.js

### PR DESCRIPTION
**Pull request by Frontrider**
added prompts

### CODE REVIEW

File: src/prompts/neuro.js [hunk 1]
```diff
@@ -42,9 +42,18 @@ export default [
   'invents new nickname',
   'whats *emote*?',
   'No~',
-  'swear word',
+  'self censors',
   'FUN AI',
   'admits to murder',
   'rants',
-  'new clone'
+  'new clone',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:49]: What does this mean exactly? She restarts?

File: src/prompts/neuro.js [hunk 2]
```diff
@@ -42,9 +42,18 @@ export default [
...
+  
+  //eg: "I'm from michigan"
+  'claims to not be british',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:52]: She admits she is American, don't think this prompt makes much sense

**Review by cjmaxik — CHANGES_REQUESTED** [review]
Thank you for the prompts. I have some feedback for you here. 

Also, please remove comments from the file, you can comment on prompts in the pull request.

File: src/prompts/neuro.js (hunk 1 above)
Comment (Frontrider) [reply src/prompts/neuro.js:49]: That is github making up a change, that line was already there.

File: src/prompts/neuro.js (hunk 2 above)
Comment (Frontrider) [reply src/prompts/neuro.js:52]: Kinda just intends to be a "wider version" of that in case she decides to say something else.
Is debatable.

Comment (cjmaxik) [reply src/prompts/neuro.js:50]: Maybe something like this?
```suggestion
  'I am <nation>',
```

File: src/prompts/neuro.js (hunk 1 above)
Comment (Frontrider) [reply src/prompts/neuro.js:49]: It is there, because I added the comma to the end of the line.

File: src/prompts/neuro.js (hunk 2 above)
Comment (Frontrider) [reply src/prompts/neuro.js]: Better.

**Comment by cjmaxik** [comment]
Thank you. heartheartheartheartheart

**Pull request merged**